Macbook-setup.md

```diff
@@ -0,0 +1,170 @@
+# Mac - new system setup instructions
+
+These instructions describe software installations for MacOS as of 6/24/2021 and are based on an Intel-based Mac running Big Sur (macOS v11.4). Any problems with this process may result from a different architecture, OS version, or newer versions of the software being installed.
+
+_Note: MacOS uses Z shell (zsh) as the default shell. If you prefer bash, you can make that the default shell under System Preferences > Users and Groups (unlock the lock icon to make changes) > right click your user, select 'Advanced Options' and change 'Login shell' to /bin/bash. Then use ~/.bashrc instead of ~/.zshrc throughout this setup._
```

Comment:
You can also just use `chsh` from the commandline to change your shell. May want to mention when you do that you'll get a scary warning every time you open Terminal saying Bash is deprecated; add `export BASH_SILENCE_DEPRECATION_WARNING=1` to .bashrc/.bash_profile to shut that up.